src/app/common/data_processing.py

```diff
@@ -34,3 +36,27 @@ def format_ncbi_summary() -> None:
         "Status",
     ]
     st.session_state[NCBI_DF] = st.session_state[NCBI_DF].reindex(columns=column_order)
+
+
+def get_top_organisms_counts() -> str:
+    """Get the top N organisms and their counts in the session's NCBI DataFrame.
+
+    Returns:
+        str: A formatted string containing the top N organisms and their counts, separated by two newlines ("\n\n").
+    """
+    # Calculate counts of species and sort by counts in descending order
+    top_n = 15
```

Comment:
I think it would be better to make this an argument for the function and set 15 as default, as maybe in the future maybe you'd want top 10 or top 5!

Reply:
Yes, I though about adding a parameter but it wasn't a strong requirement so I went ahead with no parameters. But, I have updated the function. Thank you for suggesting! :)